spec/services/articles/feed_spec.rb

```diff
@@ -197,6 +197,36 @@
     end
   end
 
+  describe "#more_tag_weight_randomized_at_end_experiment" do
+    let!(:new_story) { create(:article, published_at: 10.minutes.ago, score: 10) }
+    let(:stories) { feed.more_tag_weight_randomized_at_end_experiment }
+
+    it "includes stories" do
+      expect(stories).to include(old_story)
+      expect(stories).to include(new_story)
+    end
+  end
+
+  describe "#more_experience_level_weight_randomized_at_end_experiment" do
+    let!(:new_story) { create(:article, published_at: 10.minutes.ago, score: 10) }
+    let(:stories) { feed.more_experience_level_weight_randomized_at_end_experiment }
+
+    it "includes stories" do
+      expect(stories).to include(old_story)
+      expect(stories).to include(new_story)
+    end
+  end
+
+  describe "#more_comments_randomized_at_end_experiment" do
+    let!(:new_story) { create(:article, published_at: 10.minutes.ago, score: 10) }
+    let(:stories) { feed.more_comments_randomized_at_end_experiment }
+
+    it "includes stories" do
+      expect(stories).to include(old_story)
+      expect(stories).to include(new_story)
```

Comment:
How valuable are these tests? Perhaps we should exempt these experiment methods from CodeClimate's test coverage percentage check

Reply:
Could extract this into a shared example that loops through each experiment and ensures it does not error and returns an article.

Reply:
I think that makes sense. I feel like there is some regression test benefit to ensure these keep responding with articles, but I'll refactor into a single simpler spec.